packages/components/modal/src/modal.css

```diff
@@ -40,6 +40,10 @@ main {
 	padding: var(--ts-unit);
 }
 
+main.no-paddings {
```

Comment:
We didn't decided before, how we want to write selectors for cases like this! Are we going to use the `host` like the following :
```css
:host([no-padding]) {
```
or do the way that you did. 

IMO using the host selector is a bit better and we can keep the property for documentation reason.

Reply:
Good point!
I changed to the `host` attribute selector, I also think this is a better approach for such cases.